### PR TITLE
added pull request template

### PR DESCRIPTION
**Pull request by vitorgalvao**
Refs https://github.com/caskroom/homebrew-cask/issues/19557.

Pinging @caskroom/maintainers.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Instructions:
+
+- Delete every part of this placeholder text irrelevant to your submission, including these instructions.
+- `{{cask_file}}` is used a placeholder to the actual cask you’re submitting/editing (if applicable).
+- If there’s a checkbox you can’t complete for any reason, that’s OK. Just explain in detail why you weren’t able to do so.
+
+### Changes to a cask
+#### Editing an existing cask
+
+- [ ] Commit message includes cask’s name.
```
Comment (adidalal) [inline .github/PULL_REQUEST_TEMPLATE.md]: (and new version)

**Comment by adidalal** [comment]
Maybe add a "squashed commits (if necessary)"

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (vitorgalvao) [inline .github/PULL_REQUEST_TEMPLATE.md]: Fixed.

**Comment by vitorgalvao** [comment]
> Maybe add a "squashed commits (if necessary)"

Unsquashed commits is usually a problem after we request changes, not before (which is when this must be filled).

**Comment by vitorgalvao** [comment]
Merging. It can be amended as we see what works and doesn’t.

**Pull request merged**